utils/default-helpers/src/main/scala/pl/touk/nussknacker/engine/util/functions/collection.scala

```diff
@@ -27,25 +29,31 @@ trait CollectionUtils extends HideToString {
 
   import CollectionUtils._
 
-  @Documentation(description = "Concatenates two lists")
+  @Documentation(description = "Concatenates lists")
   @GenericType(typingFunction = classOf[ListAdditionTyping])
+  @varargs
   def concat[T](
       @ParamName("list1") list1: java.util.List[T],
-      @ParamName("list2") list2: java.util.List[T]
-  ): java.util.List[T] =
-    (list1.asScala.toList ++ list2.asScala).asJava
+      @ParamName("list2") list2: java.util.List[T]*
+  ): java.util.List[T] = {
+    val result = new java.util.ArrayList[T](list1.size() + list2.map(_.size()).sum)
+    result.addAll(list1)
+    list2.foreach(result.addAll)
+    result
+  }
 
   @Documentation(description =
-    "Merges two maps. Values in the first map will be overwritten with values from the second map if keys are the same"
+    "Merges maps. Values in the first map will be overwritten with values from the other one if keys are the same"
   )
-  @GenericType(typingFunction = classOf[MapMergeTyping])
+  @GenericType(typingFunction = classOf[VarargsMapMergeTyping])
+  @varargs
   def merge[K, V](
       @ParamName("map1") map1: java.util.Map[K, V],
-      @ParamName("map2") map2: java.util.Map[K, V]
+      @ParamName("map2") map2: java.util.Map[K, V]*
```

Comment:
Why not: 
```
@ParamName("maps") maps: java.util.Map[K, V]*
```
???

Reply:
Done

Reply:
I mean do we need two params? Can't we use only vargs:

```
@ParamName("maps") maps: java.util.Map[K, V]*
```
??

Reply:
The same with concat helper

Reply:
IMHO it is a simple way to ensure that user pass at least one collection ;)